### PR TITLE
Created TAC Minutes & Supporting Docs For 2023-12-14

### PR DESCRIPTION
**Pull request by Riaankl**

### CODE REVIEW

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+### Welcome
+
+* Dan Middleton (DM) opened the call at 7:05am PT
+* DM welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+* Ben Sternthal (BS) and Riaan Kleinhans (RK) assisted with the meeting minutes
+* DM reviewed the agenda
+
+### Roll Call / Attendance
+
+Corum was achieved with 4 voting representatives present
```
Comment (dcmiddle) [inline TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md:43]:
```suggestion
Quorum was achieved with 4 voting representatives present
```

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+## Project Proposal Coconut SVSM 
+
+Jorg Rodel (JR) proceeds to answer technical questions about the Coconut SVCM project.
+
+A vote on moving forward with Coconut SVSM was called for.
+Arm votes in favor
+Intel votes in favor
+Google vote in favor
+Microsoft votes in favor and offer to be a mentor to the project
+Readhat vote in favor
```
Comment (dcmiddle) [inline TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md:132]:
```suggestion
Redhat vote in favor
```

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+[Create Linux Kernel SIG #196](https://github.com/confidential-computing/governance/pull/196)
+
+DM discussed the content of the github issue.
+
+Voting:
+Intel votes in favor
+Google vote in favor
+Microsoft votes in favor 
+Readhat vote in favor
```
Comment (dcmiddle) [inline TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md:150]:
```suggestion
Redhat vote in favor
```

**Review by dcmiddle — APPROVED** [review]

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+* Accenture - Giuseppe Giordano 
+* Arm - Nathaniel McCallum  / Michael Lu
+* Google - Cfir Cohen  / Catalin Sandu* 
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton* (TAC chair)  / Simon Johnson
+* Meta - Eric Northup / Shankaran Gnanashanmugam
+* Microsoft - Alec Fernandez* 
+* Red Hat - Lily Sturmann  / Yash Mankad* 
+
+   " + " indicates attending
```
Comment (yashkmankad) [inline TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md:58]: Looks like " * " indicates attending :)

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+## Project Proposal Coconut SVSM 
+
+Jorg Rodel (JR) proceeds to answer technical questions about the Coconut SVCM project.
+
+A vote on moving forward with Coconut SVSM was called for.
+Arm votes in favor
+Intel votes in favor
+Google vote in favor
+Microsoft votes in favor and offer to be a mentor to the project
+Redhat vote in favor
```
Comment (yashkmankad) [inline TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md:132]: Red Hat....if not too much trouble

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+
+[Create Linux Kernel SIG #196](https://github.com/confidential-computing/governance/pull/196)
+
+DM discussed the content of the github issue.
+
+Voting:
+Intel votes in favor
+Google vote in favor
+Microsoft votes in favor 
+Redhat vote in favor
```
Comment (yashkmankad) [inline TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md:150]: Red Hat

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+
+Benjamin Sternthal (BS) spoke about optimizing the project progression process and eliminating projects, creating a charter as a first step. BS also shared the [tracking dashboard](https://github.com/orgs/confidential-computing/projects/2/views/1) they are now using for projects.
+
+Riaan Kleinhans (RK) introduced the LF Voting tool available for the TAC and other committees. Riaan or Ben can create and manage the votes as it is shared with them.
+
+
+
+## Project Proposal Coconut SVSM 
+
+Jorg Rodel (JR) proceeds to answer technical questions about the Coconut SVCM project.
```
Comment (yashkmankad) [inline TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md:125]: Jorg's last name is spelled Roedel

https://www.suse.com/c/author/joro/

**Review by yashkmankad — APPROVED** [review]
Minor changes.

But don't want to block merging, so approving.

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+
+#### Other Attendees
+
+* Bokdeuk Jeong (Samsung)
+* Chris Ramming (Broadcom)
+* Dan Willians (Intel)
+* Drasko Draskovic (Ultraviolet)
+* Eric Voit (Cisco)
+* Jiewen Yao (Intel)
+* Jorg Rodel (SUSE)
```
Comment (yashkmankad) [inline TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md:68]: Jorg Roedel....here as well

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md (hunk 7 above)
Comment (yashkmankad) [reply TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md:125]: Also ... 'Coconut SVSM'

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md (hunk 4 above)
Comment (Riaankl) [reply TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md]: Updated

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md (hunk 5 above)
Comment (Riaankl) [reply TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md]: Updated

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md (hunk 6 above)
Comment (Riaankl) [reply TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md]: Updated

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md (hunk 8 above)
Comment (Riaankl) [reply TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md]: Updated

File: TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md (hunk 7 above)
Comment (Riaankl) [reply TAC/Meetings/2023/2023-12-14/TAC Minutes 2023-12-14.md]: Updated

**Pull request merged**